libretroshare/src/turtle/p3turtle.cc

```diff
@@ -92,11 +88,11 @@ static const uint32_t TURTLE_SEARCH_RESULT_MAX_HITS_DEFAULT= 100 ;		/// default
 
 static const float depth_peer_probability[7] = { 1.0f,0.99f,0.9f,0.7f,0.6f,0.5,0.4f } ;
 
-static const int TUNNEL_REQUEST_PACKET_SIZE             = 50 ;
+static const float TUNNEL_REQUEST_PACKET_SIZE           = 50.0f ;
```

Comment:
This doesn't seems like it should be a float, maybe the error is elsewhere where this constant is used

Reply:
agreed: we may compute some float value elsewhere that is compared to this const. Would be better to cast the other thing to int, or the const to float only when it is used.

Reply:
This const is used only with float values.
So why cast it to float and not defined it to float at start?

Reply:
Then maybe the name of the constant is misleading? Calling it `*_SIZE` one expect it to be size in bytes or similar thing

Reply:
it should be a number of bytes. Phenom's option is best: just cast it to float during comparison.

Reply:
Perhaps I've explained badly.
Cast was done before (old style) during comparison. But as this value only used with float operators, why don't define it as float?
Look at [libretroshare/src/turtle/p3turtle.cc:1749](https://github.com/RetroShare/RetroShare/blob/master/libretroshare/src/turtle/p3turtle.cc#L1749)

Reply:
In this case I would rename it as `TUNNEL_REQUEST_PACKET_SIZE_APPROX` and keep it as float

Reply:
Done.